index.d.ts

```diff
@@ -1466,7 +1470,7 @@ declare namespace Eris {
     getGuildDiscovery(guildID: string): Promise<DiscoveryMetadata>;
     /** @deprecated */
     getGuildEmbed(guildID: string): Promise<Widget>;
-    getGuildIntegrations(guildID: string): Promise<GuildIntegration[]>;
+    getGuildIntegrations(guildID: string, options: GetGuildIntegrationsOptions): Promise<GuildIntegration[]>;
```

Comment:
Second param should be optional 
```suggestion
    getGuildIntegrations(guildID: string, options?: GetGuildIntegrationsOptions): Promise<GuildIntegration[]>;
```